eo-parser/src/main/resources/XMIR.xsd

```diff
@@ -83,7 +83,7 @@ SOFTWARE.
       </xs:documentation>
     </xs:annotation>
     <xs:restriction base="xs:string">
-      <xs:pattern value="[0-9]+|@|[a-z]\S*"/>
+      <xs:pattern value="α[0-9]+|@|[a-z][^ \n\t\r]*"/>
```

Comment:
@maxonfjvipon here, `\S` means anything except space, maybe it's better to keep it?